src/test_elasticsearch_server.py

```diff
@@ -4,13 +4,17 @@
 
 
 def search(query):
-    e = Elasticsearch()
+    e = Elasticsearch([{'host': 'localhost', 'port': 9200}])
     p = {"query": {"query_string": {"query": query}}}
     response = e.search(index="wikidata_en", body=json.dumps(p))
     id_labels = {}
     if response:
         for hit in response["hits"]["hits"]:
-            label = hit["_source"]["schema_name"]
+            try:
+                #same entity have schema name missing
+                label = hit["_source"]["schema_name"]
+            except Exception as e:
+                continue
```

Comment:
Wouldn't an exception be useful here, as the aim is to test ES?

Reply:
Debatable, depends in what way you want to test, if i want to query elasticsearch without it crashing all the time...